File: Advanced DSA/01. Time Complexity/04. Interesting Array/Solution.cpp
string Solution::solve(vector<int> &A) {
    int cummulative_xor = 0;
    for (int i = 0; i < A.size(); ++i)
        cummulative_xor ^= A[i];
    return (cummulative_xor & 1) ? "No" : "Yes";
}

File: Advanced DSA/01. Time Complexity/04. Interesting Array/Solution.java
public class Solution {
    public String solve(ArrayList<Integer> A) {
        int cummulative_xor = 0;
        for (int i = 0; i < A.size(); ++i)
            cummulative_xor ^= A.get(i);
        return ((cummulative_xor & 1) == 1) ? "No" : "Yes";
    }
}
